### PR TITLE
Run Call

### PR DESCRIPTION
**Pull request by waigani**
Remove Search, Review and Codemod calls. Add a generic Run call.

### CODE REVIEW

File: flow/flow.proto [hunk 1]
```diff
@@ -36,13 +36,43 @@ option objc_class_prefix = "CLW";
...
+message Reply {
+
+    // payload contains any data returned by the executed Flow.
+    google.protobuf.Any payload = 1;
+
+    // error is any error found during the execution of the Flow.
+    string error = 2;
+}
+
+message Value {
```
Comment (emersonwood) [inline flow/flow.proto]: Is this being used at all?

File: flow/flow.proto [hunk 2]
```diff
@@ -36,13 +36,43 @@ option objc_class_prefix = "CLW";
...
+        string stringProp = 1;
+        bool boolProp = 2;
+        int64 int64Prop = 3;
+        float floatProp = 4;    
+    }
+}
+
+
 // The request message containing the files or directories to review.
 message ReviewRequest {
```
Comment (emersonwood) [inline flow/flow.proto]: Shouldn't these be removed as well?

File: flow/client/client.go [hunk 3]
```diff
@@ -21,18 +21,18 @@ func NewFlowClient(conn *grpc.ClientConn) *FlowClient {
 	}
 }
 
-func (f *FlowClient) Review(ctx context.Context, req *flow.ReviewRequest) (chan *flow.Issue, chan error, error) {
-	stream, err := f.pbClient.Review(ctx, req)
+func (f *FlowClient) Run(ctx context.Context, req *flow.Request) (chan *flow.Reply, chan error, error) {
```
Comment (emersonwood) [inline flow/client/client.go]: I'm wondering if it's better to return a channel of proto any? That way the data is completely separated from the reply. Any errors on the reply can be sent on the error channel.

File: flow/flow.proto (hunk 2 above)
Comment (waigani) [reply flow/flow.proto]: We want to remove them eventually, it's just a lot more work to refactor everywhere these structs are used in platform and lingo.

**Pull request merged**